registry/docker-entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+# In production, we mount the registry TLS certificate
+# into /usr/local/share/ca-certificates. This completes
+# the certificate installation so we can trust the registry.
+if [ "$UPDATE_CA_CERTS" = "true" ]; then
+echo "Running update-ca-certificates"
+update-ca-certificates
+fi
```

Comment:
if [ "$UPDATE_CA_CERTS" ]; then => this also works @danielhoherd let me know on this

Reply:
Seems fine.

Reply:
We need to change this logic before we merge this if we are looking at the value of the variable instead of only if it is defined.

Reply:
```sh
if [ "$UPDATE_CA_CERTS" = "true" ] || [ "$UPDATE_CA_CERTS" = "True" ] ; then
echo "Running update-ca-certificates"
fi
```

our purpose is to run this command when privateCA is enabled